compute_reservation/main.tf

```diff
@@ -3,7 +3,6 @@
 resource "google_compute_reservation" "gce_reservation_local" {
   name = "gce-reservation-local"
   zone = "us-central1-c"
-  project = "my-project-name"
```

Comment:
Add?

```
data "google_project" "project" {
}
```

Reply:
Similar to above, we can continue the conversation there.